### PR TITLE
Cars

### PR DESCRIPTION
**Pull request by John-athan14**

### CODE REVIEW

**Review by Caroline-CD — DISMISSED** [review]
OK for me :100:

File: src/Components/VéhiculeCard/index.jsx [hunk 1]
```diff
@@ -1,13 +1,18 @@
 import PropTypes from 'prop-types';
-import { SVéhiculeCard, Btn, Title, Image } from './style';
+import { Link } from 'react-router-dom';
+import { SVéhiculeCard, Btn, Image, Text } from './style';
 
 export default function VéhiculeCard({ model, image, price }) {
   return (
     <SVéhiculeCard>
```
Comment (Hacene010) [inline src/Components/VéhiculeCard/index.jsx]: No "é" in english

File: src/Pages/CarsDetails/index.jsx [hunk 2]
```diff
@@ -45,91 +39,67 @@ export default function Configuration() {
...
+            <div className="ColText">
+              -46% d&apos;émission de GAES* par rapport à l&apos;achat d&apos;un
+              véhicule neuf*
             </div>
-          </Contour>
-          <Contour>
-            <div className="FullBattery">
+          </Col>
+          <Col>
+            <div className="ColLogo">
```
Comment (Hacene010) [inline src/Pages/CarsDetails/index.jsx]: className in camelCase plz

**Review by Hacene010 — CHANGES_REQUESTED** [review]

**Review by Caroline-CD — APPROVED** [review]
Ok :+1:

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**